examples/example_canister.mo

```diff
@@ -0,0 +1,58 @@
+import PT "../src";
+import Time "mo:base/Time";
+import Int "mo:base/Int";
+import Text "mo:base/Text";
+
+module TinyHttp {
+  public type Request = {
+    method : Text;
+    url : Text;
+    headers : [(Text, Text)];
+    body : Blob;
+  };
+  public type Response = {
+    status_code : Nat16;
+    headers : [(Text, Text)];
+    body : Blob;
+  };
+
+  public func render400() : Response = {
+    status_code : Nat16 = 400;
+    headers : [(Text, Text)] = [];
+    body : Blob = "Invalid request";
+  };
+  public func renderPlainText(text : Text) : Response = {
+    status_code = 200;
+    headers = [("content-type", "text/plain")];
+    body = Text.encodeUtf8(text);
+  };
+};
```

Comment:
Could be useful to put this into a separate file so users can just copy it into their project.

Reply:
Done